src/anyp/PortCfg.h

```diff
@@ -72,17 +78,7 @@ class PortCfg : public RefCountable
     /// TLS configuration options for this listening port
     Security::ServerOptions secure;
 
-#if USE_OPENSSL
-    char *sslContextSessionId; ///< "session id context" for secure.staticSslContext
-    bool generateHostCertificates; ///< dynamically make host cert for sslBump
-    size_t dynamicCertMemCacheSize; ///< max size of generated certificates memory cache
-
-    Security::CertPointer signingCert; ///< x509 certificate for signing generated certificates
-    Ssl::EVP_PKEY_Pointer signPkey; ///< private key for sighing generated certificates
-    Ssl::X509_STACK_Pointer certsToChain; ///<  x509 certificates to send with the generated cert
-    Security::CertPointer untrustedSigningCert; ///< x509 certificate for signing untrusted generated certificates
-    Ssl::EVP_PKEY_Pointer untrustedSignPkey; ///< private key for signing untrusted generated certificates
-#endif
+    size_t dynamicCertMemCacheSize = 4*1024*1024; ///< max size of generated certificates memory cache (4 MB default)
```

Comment:
It is not actually a TLS/SSL thing. It is just a per-port cache structure managed through SMP and Runners regardless of whether TLS/SSL is even used on the port. That seems like a bit of a design mistake, but not one I'm fixing in this PR.